gtsfm/two_view_estimator.py

```diff
@@ -59,7 +59,7 @@ def __init__(
         bundle_adjust_2view: bool,
         eval_threshold_px: float,
         bundle_adjust_2view_maxiters: int = 100,
-        ba_reproj_error_thresh: float = 0.5,
+        ba_reproj_error_thresh: List[float] = [0.5],
```

Comment:
nit: would prefer to rename this to `ba_reproj_error_thresholds` since I think of `thresh` as singular